src/components/OrgUnitsTreeWithExtraOptions.js

```diff
@@ -32,14 +34,31 @@ export const OrgUnitsTreeWithExtraOptions = props => (
             </span>
             <div
                 id="extra-options"
-                style={getExtraOptionsStyle(props.showOptions)}
+                className={getExtraOptionsClassName(props.showOptions)}
```

Comment:
I'm really curious how this works.. I just tested it and it works, but I still don't understand how it's done technically

Reply:
It just returns the string "show-options" or "hide-options". So it's . dynamic className